### DISCUSSION TITLE
[灌水] 绳网小助手更新也太快了

### DISCUSSION THREAD

**Discussion by ZMJ2333** [share121/inter-knot#1460] [其他]
![微信图片_20240811190259](https://github.com/user-attachments/assets/95c9ff30-9360-489a-978e-be8ebb93f119)
我每次刷新都更新一次（还是太巧了？）
好奇作者的迭代速度

**Comment by 3112024746** [comment]
我也是刚开始更新的

**Comment by ZhuYuan-148** [comment]
能感觉到绳网管理员在努力提升用户体验。

**Comment by ZMJ2333** [comment]
回警察同志
 ### Affirmative！

**Comment by Dave-12138** [comment]
个人开发是这样的，做完一个功能，发布更新，刚发完发现又多了个新需求，开始做，做完再发更新，没有固定的更新周期——开发者会觉得新功能/bug修复不应该越快上线越好吗，同时也觉得“用户肯定也这么想的”。


_所以个人开发有个版本控制已经很不错了，更新周期稳定太勉强（_

**Comment by ZMJ2333** [comment]
回楼上，只是好奇作者迭代这么快就不休息的么

**Reply by qiuledadada** [reply]
有没有可能……作者就是今天休息才来更新的

**Reply by ZMJ2333** [reply]
啊？

**Comment by fairy-B** [comment]
是啊，今天更新了两次啊！

**Reply by ZMJ2333** [reply]
好像不止吧？

**Reply by Dave-12138** [reply]
今天更了4次，1.3、1.4、1.5、1.6

**Comment by lllliufeng** [comment]
好快

**Comment by GanlinSama** [comment]
今天两次比较大的改动应该就是等级和浏览量了，虽然都是半成品，但是可以看出来作者的诚意

**Comment by 290678** [comment]
积极的up主为什么没人关注呢

**Comment by 1451374815** [comment]
还好吧，对我来说好像就一天一次

**Comment by XxnjiIAJ** [comment]
今天每次上线都有更新
